.php_cs.cache

```diff
@@ -0,0 +1 @@
s:13:"join_function";i:16;s:20:"line_after_namespace";i:17;s:8:"linefeed";i:18;s:11:"list_commas";i:19;s:17:"long_array_syntax";i:20;s:19:"lowercase_constants";i:21;s:18:"lowercase_keywords";i:22;s:21:"method_argument_space";i:23;s:30:"multiline_array_trailing_comma";i:24;s:12:"multiple_use";i:25;s:31:"namespace_no_leading_whitespace";i:26;s:15:"new_with_braces";i:27;s:22:"newline_after_open_tag";i:28;s:34:"no_blank_lines_after_class_opening";i:29;s:28:"no_empty_lines_after_phpdocs";i:30;s:15:"object_operator";i:31;s:16:"operators_spaces";i:32;s:11:"ordered_use";i:33;s:11:"parenthesis";i:34;s:15:"php_closing_tag";i:35;s:13:"phpdoc_indent";i:36;s:17:"phpdoc_inline_tag";i:37;s:16:"phpdoc_no_access";i:38;s:22:"phpdoc_no_empty_return";i:39;s:17:"phpdoc_no_package";i:40;s:13:"phpdoc_params";i:41;s:13:"phpdoc_scalar";i:42;s:17:"phpdoc_separation";i:43;s:24:"phpdoc_short_description";i:44;s:17:"phpdoc_to_comment";i:45;s:11:"phpdoc_trim";i:46;s:18:"phpdoc_type_to_var";i:47;s:23:"phpdoc_var_without_name";i:48;s:13:"pre_increment";i:49;s:4:"psr0";i:50;s:24:"remove_leading_slash_use";i:51;s:25:"remove_lines_between_uses";i:52;s:6:"return";i:53;s:13:"self_accessor";i:54;s:9:"short_tag";i:55;s:30:"single_array_no_trailing_comma";i:56;s:34:"single_blank_line_before_namespace";i:57;s:25:"single_line_after_imports";i:58;s:12:"single_quote";i:59;s:23:"spaces_before_semicolon";i:60;s:11:"spaces_cast";i:61;s:21:"standardize_not_equal";i:62;s:14:"ternary_spaces";i:63;s:15:"trailing_spaces";i:64;s:17:"trim_array_spaces";i:65;s:22:"unary_operators_spaces";i:66;s:10:"unused_use";i:67;s:10:"visibility";i:68;s:16:"whitespacy_lines";}s:6:"hashes";a:21:{s:29:"Definition/PaginatorAware.php";i:2357445140;s:38:"Definition/PaginatorAwareInterface.php";i:758858995;s:51:"DependencyInjection/Compiler/PaginatorAwarePass.php";i:2053279218;s:59:"DependencyInjection/Compiler/PaginatorConfigurationPass.php";i:1282080587;s:37:"DependencyInjection/Configuration.php";i:4185120738;s:45:"DependencyInjection/KnpPaginatorExtension.php";i:699995047;s:20:"Helper/Processor.php";i:3753383689;s:22:"KnpPaginatorBundle.php";i:3069171862;s:32:"Pagination/SlidingPagination.php";i:2535271817;s:30:"Resources/config/paginator.xml";i:2790977602;s:35:"Resources/config/templating_php.xml";i:1127957801;s:47:"Resources/views/Pagination/filtration.html.twig";i:1986456125;s:61:"Resources/views/Pagination/foundation_v5_pagination.html.twig";i:3933865959;s:44:"Resources/views/Pagination/sliding.html.twig";i:500847863;s:50:"Resources/views/Pagination/sortable_link.html.twig";i:2988099577;s:65:"Resources/views/Pagination/twitter_bootstrap_pagination.html.twig";i:4150215577;s:68:"Resources/views/Pagination/twitter_bootstrap_v3_pagination.html.twig";i:1240184693;s:42:"Subscriber/SlidingPaginationSubscriber.php";i:1320485324;s:31:"Templating/PaginationHelper.php";i:3793346514;s:61:"Tests/DependencyInjection/Compiler/PaginatorAwarePassTest.php";i:1100239802;s:38:"Twig/Extension/PaginationExtension.php";i:4174030327;}}
```

Comment:
ok, but then we could also remove the ``setUsingCache(true)`` from the config, otherwise we have to ignore the file then in ``.gitignore`, so that you don't accidentally commit it.